apps/web/src/pages/auth/components/LoginForm.tsx

```diff
@@ -95,24 +97,39 @@ export function LoginForm({ email, invitationToken }: LoginFormProps) {
 
   return (
     <>
-      {!IS_DOCKER_HOSTED && (
+      <When truthy={!IS_DOCKER_HOSTED}>
         <>
-          <GitHubButton
-            component="a"
-            href={githubLink}
-            my={30}
-            variant="white"
-            fullWidth
-            radius="md"
-            leftIcon={<GitHub />}
-            sx={{ color: colors.B40, fontSize: '16px', fontWeight: 700, height: '50px' }}
-            data-test-id="github-button"
-          >
-            Sign In with GitHub
-          </GitHubButton>
+          <OAuth>
+            <GitHubButton
+              component="a"
+              href={githubLink}
+              my={30}
+              variant="white"
+              fullWidth
+              radius="md"
+              leftIcon={<GitHub />}
+              sx={{ color: colors.B40, fontSize: '16px', fontWeight: 700, height: '50px', marginRight: 10 }}
+              data-test-id="github-button"
+            >
+              Sign In with GitHub
+            </GitHubButton>
+            <GitHubButton
+              component="a"
+              href={googleLink}
+              my={30}
+              variant="white"
+              fullWidth
+              radius="md"
+              leftIcon={<Google />}
+              data-test-id="google-button"
+              sx={{ color: colors.B40, fontSize: '16px', fontWeight: 700, height: '50px', marginLeft: 10 }}
+            >
+              Sign In with Google
+            </GitHubButton>
```

Comment:
It doesn't make much sense to me that the component rendering the SSO for Google is called `GitHubButton`.
I probably would expect this component to look like:
```
<OAuth>
  <GitHubButton link={gitHubLink} />
  <GoogleButton token={invitationToken} />
 </OAuth>
```
With some of the logic encapsulated in the different button components.

Reply:
Totally write i probably missed this one